styles.css

```diff
@@ -0,0 +1,179 @@
+
+.tool-bar {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
+  background: #3c3a39;
+}
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT:** To make your web page responsive I suggest you update the `width: 375px;` to be `  width: 100%;`